threads-api/src/threads-api.ts

```diff
@@ -934,9 +1070,229 @@ export class ThreadsAPI {
+      signed_body: `SIGNATURE.${encodeURIComponent(JSON.stringify(data))}`,
+    }
+
+    const res = await this._toggleAuthPostRequest<any>(url, payload, options);
+    if (this.verbose) {
+      console.debug('[MUTE]', res.data);
+    }
+    return res.data.status === 'ok';
+  };
+  unmute = async (muteOptions: {postID?: string, userID?: string}, options?: AxiosRequestConfig): Promise<boolean> => {
+    const url = `${BASE_API_URL}/api/v1/friendships/unmute_posts_or_story_from_follow/`;
+    let data = {
+      _uid: this.userID,
+      _uuid: this.deviceID,
+      container_module: "ig_text_feed_timeline"
+    } as {
+      media_id?: string;
+      _uid: string;
+      _uuid: string;
+      container_module: string;
+      target_posts_author_id?: string;
+    };
+
+    if (!muteOptions.postID && !muteOptions.userID) {
+      throw new Error('Post ID or User ID is required');
+    }
+
+    if (muteOptions.postID) {
+      data.media_id = String(muteOptions.postID);
+    }
+    if (muteOptions.userID) {
+      data.target_posts_author_id = String(muteOptions.userID);
+    }
+
+    const payload = {
+      signed_body: `SIGNATURE.${encodeURIComponent(JSON.stringify(data))}`,
+    }
+
+    const res = await this._toggleAuthPostRequest<any>(url, payload, options);
+    if (this.verbose) {
+      console.debug('[UNMUTE]', res.data);
+    }
+    return res.data;
+  };
+  block = async (userID: string, options?: AxiosRequestConfig): Promise<boolean> => {
+    const url = `${BASE_API_URL}/api/v1/friendships/block/${userID}/`;
+    let data = {
+      surface: 'ig_text_feed_timeline',
+      is_auto_block_enabled: true,
+      user_id: userID,
+      _uid: this.userID,
+      _uuid: this.deviceID
+    }
+
+    const payload = {
+      signed_body: `SIGNATURE.${encodeURIComponent(JSON.stringify(data))}`,
+    }
+
+    const res = await this._toggleAuthPostRequest<any>(url, payload, options);
+    if (this.verbose) {
+      console.debug('[MUTE]', res.data);
+    }
+    return res.data.status === 'ok';
+  };
+  unblock = async (userID: string, options?: AxiosRequestConfig): Promise<boolean> => {
+    const url = `${BASE_API_URL}/api/v1/friendships/unblock/${userID}/`;
+    let data = {
+      user_id: userID,
+      _uid: this.userID,
+      _uuid: this.deviceID,
+      container_module: 'ig_text_feed_timeline'
+    }
+
+    const payload = {
+      signed_body: `SIGNATURE.${encodeURIComponent(JSON.stringify(data))}`,
+    }
+
+    const res = await this._toggleAuthPostRequest<any>(url, payload, options);
+    if (this.verbose) {
+      console.debug('[MUTE]', res.data);
+    }
+    return res.data.status === 'ok';
+  };
+
+  getNotifications: PaginationNotificationsQuerier<GetNotificationsPaginatedResponse> = async (
+    filter,
+    pagination,
+    options = {},
+  ): Promise<GetNotificationsPaginatedResponse> => {
+
+    let params: GetNotificationsOptions = {
+			feed_type: 'all',
+			mark_as_seen: false,
+			timezone_offset: -25200,
```

Comment:
Can we have the `timezone_offset` from the user locale, as in #50?

https://github.com/junhoyeo/threads-api/blob/200ad48d33dc8afe6d14a6e627ac36b1fe711820/threads-api/src/threads-api.ts#L1328-L1333

We should manage this value as an memoized getter...